aspnetcore/blazor/fundamentals/routing.md

```diff
@@ -33,77 +44,96 @@ Interactive routing also prevents prerendering because new page content isn't re
 
 :::moniker-end
 
-## Route templates
+## ASP.NET Core endpoint routing integration
+
+*This section applies to Blazor Web Apps and Blazor Server apps operating over a circuit.*
```

Comment:
The content for .NET 8+ really only covers endpoint routing integration for Blazor Web Apps. The Blazor Server content is only shown for earlier .NET versions. I think this line needs to be within the version range monikers and adjusted accordingly.